03-extract_events.py

```diff
@@ -38,8 +38,8 @@ def run_events(subject):
 
         if config.plot:
             # plot events
-            figure = mne.viz.plot_events(events)
-            figure.show()
+            fig = mne.viz.plot_events(events, show=True)
```

Comment:
you should be using:

`mne.viz.plot_events(events, sfreq=raw.info['sfreq'], first_samp=raw.first_samp)`